content/applications/websites/ecommerce/managing_products/variants.rst

```diff
@@ -1,110 +1,88 @@
-=======================
-Manage product variants
-=======================
+================
+Product variants
+================
 
-Product variants are used to offer variations of the same product to your customers on the product's
-page.
+Product variants are variations such as color, material, etc. of a same product. These variations
+can differ in price and availability from the product. Product variants can either be
+:doc:`created <../../../sales/sales/products_prices/products/variants>` or
+:doc:`imported <../../../sales/sales/products_prices/products/import>`.
 
-Let's say one of your customers selects a T-shirt from your product catalog.
-
-Then, they choose the size and color they want. The various size and color combinations are known as
-product variants.
-
-Create attributes & variants
-============================
-
-In order to create various attributes and variants for your products, you first need to turn on
-*Variants* in :menuselection:`Sales --> Configuration --> Settings`.
-
-Then, select a product from the Products list (:menuselection:`Sales / Website --> Products -->
-Products`). When the product detail form opens, click on *Edit*, and go to the *Variants* tab.
-
-Here, you can add as many different attributes as you'd like by clicking on *Add a line*.
-
-These attributes appear as three different types:
+.. note::
+   :guilabel:`Product Variants` has to be enabled under :menuselection:`Configuration --> Settings`,
+   in the :guilabel:`Shop - Products` section.
 
-- Radio buttons
-- Color buttons
-- Drop-down menu
+.. seealso::
+   - :doc:`../../../sales/sales/products_prices/products/variants`
+   - :doc:`../../../sales/sales/products_prices/products/import`
 
-Several variants appear as soon as there are two values for one attribute.
+Product configurator
+====================
 
-Don't forget to hit *Save* once all variants have been entered.
+In the context of eCommerce, adding attributes and values to a product template allows for the
+addition of a **product configurator** on the product page. The customer is able to configure and
+select the product variant of their choice; or in the case of multiple attributes, combine those to
+create a specific variant.
```

Comment:
```suggestion
In the context of eCommerce, adding attributes and values to a product template allows for adding a **product configurator** on the product page. Customers are able to configure and
select the product variant of their choice; or in the case of multiple attributes, combine those to
create a specific variant.
```
I would change it to a plural as "the customer" (sg) --> of "his/her" choice

Reply:
I changed it up a bit, thanks for the suggestion! FYI, "their" is the neutral form (and therefore preferred), and works for both Sg. and Pl. :)